hphp/test/frameworks/results/mockery.expect

```diff
@@ -727,8 +739,6 @@ Mockery_MockTest::testShouldAllowMockingProtectedMethodReturnsMockInstance
 NamedMockTest::itCreatesANamedMock
 .
 NamedMockTest::itCreatesPassesFurtherArgumentsJustLikeMock
-FATAL
-NamedMockTest::itShouldThrowIfAttemptingToRedefineNamedMock
```

Comment:
Could just be a misunderstanding on my part, i.e. the .expects should have the results that will occur at present (incl. FATAL) vs me submitting without the FATAL.